src/app.ts

```diff
@@ -1,27 +1,27 @@
-import { MainPageController } from "./pages/mainPage/MainPageController"
-import { getCookie } from "./shared/API";
-import { startRouting } from "./shared/router";
-import Header from "./widgets/header/headerView";
-
+import { setCountProductInBasket } from './pages/basketPage/basketPageController';
+import { MainPageController } from './pages/mainPage/MainPageController';
+import { getCookie } from './shared/API';
+import { startRouting } from './shared/router';
+import Header from './widgets/header/headerView';
 
 export async function initializeApp(): Promise<void> {
   try {
     const headerContainer = document.createElement('div');
-    headerContainer.className = "header__wrapper";
+    headerContainer.className = 'header__wrapper';
     document.body.append(headerContainer);
     const access = getCookie('access_token');
     const shouldRender = !!access;
     Header.render(shouldRender);
     const appContainer = document.createElement('div');
-    appContainer.id = "app";
+    appContainer.id = 'app';
     document.body.append(appContainer);
-
     const savedState = localStorage.getItem('appState');
     if (!savedState) {
       appContainer.innerHTML += MainPageController();
     }
+    await setCountProductInBasket();
     startRouting();
   } catch (error) {
     throw new Error();
```

Comment:
Optional. It is better to throw your error further.
```
catch (error) {
  throw error;
}
```
So you can see you error in console while debugging.